README.md

```diff
@@ -397,6 +397,115 @@ const { hash } = await erc165SessionKeyProvider.sendUserOperation({
 });
 ```
 
+### Session Key Validator
+
+```ts
+// 1. Get the default ecdsa validator provider
+const ecdsaProvider = await ECDSAProvider.init({
+  projectId, // zeroDev projectId
+  owner,
+});
+
+// 2. Deploy the Kernel account if not already by sending an empty transaction
+let result = await ecdsaProvider.sendUserOperation({
+  target: "0xADDRESS",
+  data: "0x",
+});
+await ecdsaProvider.waitForUserOperationTransaction(result.hash as Hex);
+
+// 3. Initialize required variables
+const accountAddress = await ecdsaProvider.getAccount().getAddress();
+const selector = getFunctionSelector("execute(address, uint256, bytes, uint8)");
+const sig = getFunctionSelector(
+    "transfer(address, uint256)"
+  )
+const permissions = [
+    {
+        target: <ERC20Address>, // address of the target contract
+        valueLimit: 0, // max value the session key can use in tx
+        sig, // The function selector of the function that can be called on the target contract
+        operation: Operation.Call, // The kind of call session key can make CALL/DELEGATECALL
+        rules: [ // Parameter rules
+        {
+            condition: ParamCondition.LESS_THAN_OR_EQUAL, // The condition to check
+            offset: 32, // The offset where the param is in the calldata
+            param: pad(toHex(10000), { size: 32 }), // The value to check in condition
+        },
+        {
+            condition: ParamCondition.EQUAL,
+            offset: 0,
+            param: pad(<SPECIFIC_ADDRESS>, { size: 32 }),
+        },
+        ],
+    }
+]
+
+// 4. Initialize SessionKey Validator Provider
+const sessionKeyProvider = await SessionKeyProvider.init({
+      projectId, //ZeroDevProject
+      owner,
+      sessionKey, // Session Key Signer
+      sessionKeyData: {
+        validAfter: 0,
+        validUntil: 0,
+        permissions,
+        permissionIndex, // The index of the permission to use for the tx through this provider
```

Comment:
What is the `permissionIndex` for?  Not sure if I understand

Reply:
Yes. That's one thing I am writing util for. It will be removed

Reply:
Done ✅